### DISCUSSION TITLE
Where can I see ping results in Zwave2MQTT / ZWJSUI?

### DISCUSSION THREAD

**Discussion by matejdro** [zwave-js/zwave-js-ui#2675] [Q&A] [answered]
https://zwave-js.github.io/node-zwave-js/#/troubleshooting/network-health?id=testing-the-connection-strength mentions about ability to view information, such as power level of the node.

However, I cannot find where to see the results of the ping? There is a ping button (which seems to only print `Success zwave api call pingNode true` into the logs) and there is statistics button which shows long term statistics instead of ping results mentioned on the above page.

**Comment by kpine** [comment, marked as answer]
https://zwave-js.github.io/zwave-js-ui/#/usage/nodes_healthcheck
